data/remnant/remnant missions.txt

```diff
@@ -875,10 +879,63 @@ mission "Remnant: Technology Available"
+			`	"Now, before we arrive at <destination>, pick a pattern for yourself. Something short - a second or two at most, and something memorable to you." The prefect turns away from you to install a small device on the side of one of your bridge consoles, and connects a few wires into the panel. "Now, insert your license here, and tap your pattern on the license." He looks away as you follow his directions. "Done? Good. When we approach <destination>, you will have a few options: For a normal approach when everything is nominal, tap your pattern on the license. If there is a threat following you in - an enemy ship or some other kind of danger you want the monitors to watch for, pinch or squeeze the license instead. And if you have a problem onboard, such as having been captured and forced to fly the ship in tap the license in a random pattern. Lastly, if you are having trouble landing due to injury or damage to the ship, just press briefly on the license. If you don't touch the license at all, the monitors will assume that you are either incapacitated or otherwise not in command of your ship. We will still attempt to let you land safely and provide assistance, but will also be standing ready to destroy you and block your approach if we think you are becoming a threat."`
+	on complete
+		conversation
+			`	As you enter the upper atmosphere of <destination>, the prefect gestures toward the small bracket he installed on your console. You insert your license, and tap the pattern you created. He checks something on his commlink and then nods. "Your license and clearance have been accepted, Captain <first>. Carry on with landing as normal."  You proceed with the approach and landing, with the prefect pointing out noteworthy features along the way. He also points out various approach paths, including prefered routes to take if you are being chased. "If you have a hostile threat on your tail, take this landing corridor; which will expose trailing ships to fire from these points. And if you have something sticking tight in your shadow, do a roll at this point or that point. That will expose them to our high-precision batteries."`
+			choice
+				`	"Thank you for the advice."`
+					goto conclusion
+				`	"I'm surprised you are telling me all this."`
+			`	"Oh, do not worry that you know too much. There are far more defences that I have not mentioned, and we are always improving them. Generally, our defences should be invisible to you - as in, they just stay out of your way and are not something you pay attention to."`
+			label conclusion
+			`	"Your approach goes smoothly, and you glide past a number of anti-air batteries that are methodically covering the sky around you. Once the <ship> has settled safely to the ground, the prefect pulls the license from the socket and hands it back to you. "At the risk of sounding like a parody: Keep it secret, keep it safe. May the Embers burn bright for you." And with that, the old prefect walks off your ship.`
```

Comment:
"Parody"? Sounds kinda out of place for the Remnant.

Reply:
Mmm, not sure what you mean. They have access to the full "common" language, no reason why a word like "parody" couldn't be in their vocabulary.